env-variables.html

```diff
@@ -1427,13 +1463,295 @@ <h5>CCL_ENABLE_SYCL_KERNELS<a class="headerlink" href="#ccl-enable-sycl-kernels"
 </div>
 </section>
 </section>
+<section id="id4">
+<h4>ALLGATHER/ALLGATHERV<a class="headerlink" href="#id4" title="Permalink to this headline">#</a></h4>
+<section id="ccl-sycl-allgatherv-tmp-buf">
+<h5>CCL_SYCL_ALLGATHERV_TMP_BUF<a class="headerlink" href="#ccl-sycl-allgatherv-tmp-buf" title="Permalink to this headline">#</a></h5>
+<p><strong>Syntax</strong></p>
+<div class="highlight-cpp notranslate"><div class="highlight"><pre><span></span><span class="n">CCL_SYCL_ALLGATHERV_TMP_BUF</span><span class="o">=&lt;</span><span class="n">value</span><span class="o">&gt;</span>
+</pre></div>
+</div>
+<p><strong>Arguments</strong></p>
+<table class="colwidths-given table table-left">
+<colgroup>
+<col style="width: 33%" />
+<col style="width: 67%" />
+</colgroup>
+<tbody>
+<tr class="row-odd"><td><p>&lt;value&gt;</p></td>
+<td><p>Description</p></td>
+</tr>
+<tr class="row-even"><td><p><code class="docutils literal notranslate"><span class="pre">1</span></code></p></td>
+<td><p>Uses a persistent temporary buffer to perform the <code class="docutils literal notranslate"><span class="pre">ALLGATHER</span></code>/ <code class="docutils literal notranslate"><span class="pre">ALLGATHERV</span></code>.</p></td>
+</tr>
+<tr class="row-odd"><td><p><code class="docutils literal notranslate"><span class="pre">0</span></code></p></td>
+<td><p>Performs an IPC handle exchange, avoiding copies to temporary buffers. Default value.</p></td>
+</tr>
+</tbody>
+</table>
+<p><strong>Description</strong></p>
+<p>Specifies if the <code class="docutils literal notranslate"><span class="pre">ALLGATHER</span></code>/ <code class="docutils literal notranslate"><span class="pre">ALLGATHERV</span></code> implementation should use a persistent temporary buffer or not. The implementation with temporary buffers makes the collective fully asynchronous, but adds some additional overhead due to the extra copy of the user buffer to a (persistent) temporary buffer. The current default uses Level Zero IPC to avoid the copies to the temporary buffer.</p>
```

Comment:
it somehow internally hurts me when I hear "persistent temporary" ... there are more occurrences of that, I won't mark them from now on to avoid clutter.